helm/examples/efs/README.md

```diff
@@ -0,0 +1,22 @@
+# HPCC Elastic File System (EFS) Storage
+
+## Directory contents
+
+### hpcc-efs/
+The hpcc-efs helm chart will provision a EFS Server Pod. It will also creates a Persistent Volume Claim for each of the required HPCC storage types.
+
+EFS Serve should be created on AWS EFS service first.  hpcc-efs helm chart need EFS region and id to configure the deployment. The default region is us-east-1.
+
+Example use:
+
+helm install efsstorage hpcc-efs/ --set efs.region=<efs region>  --set efs.id=<efs id>
```

Comment:
Would
   helm install efsstorage examples/efs/hpcc-efs
be better?  Assuming that the current directory is <root>/helm.

Reply:
updated